stock_intercompany_bidirectional/models/res_company.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    intercompany_out_type_id = fields.Many2one(
+        "stock.picking.type", string="Intercompany Operation Type OUT"
+    )
+    intercompany_in_type_id = fields.Many2one(string="Intercompany Operation Type IN")
+    set_done_qty = fields.Boolean(string="Set Qty Done")
```

Comment:
It seems that the variable name `set_done_qty` could be more descriptive. A name that more clearly represents this action would significantly enhance code readability and maintainability. Maybe `auto_update_qty_done` with some additional help=`...`